app/views/campaign_requests/_request_details.html.erb

```diff
@@ -2,26 +2,148 @@
+    </p>
 
-    <%= r.input :description, as: :text, label: "Campaign description", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
+    <%= r.input :call_to_action,
+                as: :text,
+                label: "Call to action",
+                required: true,
+                input_html: {:class => "input-md-6",
+                             :rows => 6,
+                             :cols => 50,
+                             :"aria-required" => true } %>
 
-    <%= r.input :call_to_action, as: :text, label: "Call to action", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
-    <%= r.input :success_measure, as: :text, label: "How will you measure success?", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
+    <%= r.input :proposed_url,
+                label: "Proposed URL (in the form of xxxxx.campaign.gov.uk)",
+                required: true, input_html: { class: "input-md-6",
+                                              :"aria-describedby" => "urlHelpBlock",
+                                              :"aria-required" => true } %>
+    <p class="help-block">
+      If applying for Campaigns Platform it is in the form of xxxxx.campaign.gov.uk, or otherwise if a microsite, standard format is  in the form of xxxxxx.gov.uk however if you require something else please state your reasons here
+    </p>
 
-    <%= r.input :proposed_url, label: "Proposed URL (in the form of xxxxx.campaign.gov.uk)", required: true, input_html: { class: "input-md-6", :"aria-describedby" => "urlHelpBlock", :"aria-required" => true } %>
+    <%= r.input :site_metadescription,
+                as: :text,
+                label: "Site metadescription (appears in search results)",
+                required: true,
+                input_html: {:class => "input-md-6",
+                             :rows => 6,
+                             :cols => 50,
+                             :"aria-required" => true,
+                             :"aria-describedby" => "seoHelpBlock" } %>
+    <p class="help-block">
+      The meta description is a short, clear sentence how you want the site described as it appears in Google search
+    </p>
 
-    <%= r.input :site_metadescription, as: :text, label: "Site metadescription (appears in search results)", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true, :"aria-describedby" => "seoHelpBlock" } %>
+    <%= r.input :cost_of_campaign, label: "Site build budget / costs (and overall campaign cost, if applicable)", required: true, input_html: { class: "input-md-6" } %>
+    <p class="help-block">
+      If you are applying for the Campaign Platform page, this is free of any build charge
+    </p>
```

Comment:
As this bit of the form is only relevant to people wanting a bespoke site, are we able to hide it for campaign sites and only show it to those choosing the bespoke site option?  It'd mean we wouldn't need this help block to explain to people choosing a campaign site that they can leave this blank?  But maybe that's intentional, as a pointer to make people choose campaign instead of bespoke?

Reply:
Will ask.

Reply:
Have asked. We're keeping it for both types of websites as it's a way for them to keep track of costs.